### PR TITLE
implement gh actions workflow

### PR DESCRIPTION
**Pull request by michaelshobbs**
ref: https://github.com/lib/pq/pull/1047#issuecomment-906062580

the `TestSSLClientCertificates` fails on postgres 13 & 12 with `ssl_test.go:269: remote error: tls: bad certificate` but the other postgres versions seem to work.

EDIT: also `TestSSLVerifyFull` fails on golang 1.17 with `ssl_test.go:104: x509: certificate relies on legacy Common Name field, use SANs instead`. From what I can tell, they removed the `GODEBUG=x509ignoreCN=0` flag.

### CODE REVIEW

**Comment by michaelshobbs** [comment]
@otan @mjibson I'm not an SSL expert by any stretch so any pointers here would be great

**Comment by otan** [comment]
i'm no i'm no export either, but my guess is those certificates have to be regenerated.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.PHONY: all root-ssl server-ssl client-ssl
```
Comment (mjibson) [inline Makefile:1]: Can you move this to the certs dir and add a comment explaining what it does?

Comment (michaelshobbs) [reply Makefile]: done 7da16d9

**Comment by otan** [comment]
this is awesome, cheers buddy

**Pull request merged**